index.html

```diff
@@ -27,7 +27,7 @@
 <div id="key"></div>
 <div id="instructions">Ctrl+Shift+Q to close</div>
 <script src="mousetrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/tone/14.4.84/Tone.js"></script>
```

Comment:
Yeah I tried to use NPM but wasn't able to immediately figure out how to use it on the front-end side. I suppose that's different from just running something locally (e.g. Electron).

If NPM is better, or if you want me to include a minified file like you did for Mousetrap, let me know.....

Reply:
Just tried parcel, it's easy to use, future PR